File: codigofonte6 (terminar tradução)/1/condicoes.py
# Condições e operadores relacionais

from cs50 import get_int

# Pede ao usuário para informar x
x = get_int("x: ")

# Pede ao usuário para informar y
y = get_int("y: ")

# Compara x e y
if x < y:
    print("x é menor do que y")
elif x > y:
    print("x é maior do que y")
else:
    print("x é igual à y")

File: codigofonte6 (terminar tradução)/1/condicoes.c
// Condicionais e operadores relacionais

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Pede ao usuário para informar x
    int x = get_int("x: ");

    // Pede ao usuário para informar y
    int y = get_int("y: ");

    // Compara x e y
    if (x < y)
    {
        printf("x é menor do que y\n");
    }
    else if (x > y)
    {
        printf("x é maior do que y\n");
    }
    else
    {
        printf("x é igual a y\n");
    }
}
